### PR TITLE
Rephrased grammar in Change Detection documentation

### PR DESCRIPTION
**Pull request by Adeniyikayodee**
Rephrased grammar in Change Detection documentation.

### CODE REVIEW

**Review by soerenmartius — APPROVED** [review]
lgtm, good job @Adeniyikayodee

File: docs/change-detection/index.md [hunk 1]
```diff
@@ -39,10 +40,11 @@ default branch or in a feature branch, and they are:
...
 parent of the `HEAD` commit and the reasoning for using it for the default
 branch is that once you merged your PR you need to apply the changes in the CI
-or locally. So if the project uses
-[non-fast-forwarded](https://git-scm.com/docs/git-merge#_fast_forward_merge)
-all commits (except first) in the default branch are merge commits, then by
-using `HEAD^` as baseref we can detect changes of the last merged code.
+or locally. If the project adopts a
+[non-fast-forward](https://git-scm.com/docs/git-merge#_fast_forward_merge)
+merge strategy, every commit—aside from the first one—on the default branch becomes a merge
+commit. Utilizing `HEAD^` as the `baseref`enables detection of modifications in the most
```
Comment (i4ki) [inline docs/change-detection/index.md:46]:
```suggestion
commit. Utilizing `HEAD^` as the `baseref` enables detection of modifications in the most
```

**Review by i4ki — DISMISSED** [review]
lgtm but have a look in the small/minor suggestion

File: docs/change-detection/index.md (hunk 1 above)
Comment (soerenmartius) [reply docs/change-detection/index.md:46]: good catch @i4ki

**Review by soerenmartius — APPROVED** [review]
lgtm

**Pull request merged**